uxkit-screenshare/src/main/java/com/voxeet/uxkit/screenshare/ScreenShareMediaPresentationView.java

```diff
@@ -0,0 +1,251 @@
+package com.voxeet.uxkit.screenshare;
+
+import android.annotation.SuppressLint;
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ScaleGestureDetector;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.voxeet.VoxeetSDK;
+import com.voxeet.android.media.MediaStream;
+import com.voxeet.android.media.stream.MediaStreamType;
+import com.voxeet.sdk.events.v2.StreamAddedEvent;
+import com.voxeet.sdk.events.v2.StreamRemovedEvent;
+import com.voxeet.sdk.events.v2.StreamUpdatedEvent;
+import com.voxeet.sdk.json.VideoPresentationPaused;
+import com.voxeet.sdk.json.VideoPresentationPlay;
+import com.voxeet.sdk.json.VideoPresentationSeek;
+import com.voxeet.sdk.json.VideoPresentationStarted;
+import com.voxeet.sdk.json.VideoPresentationStopped;
+import com.voxeet.sdk.models.Participant;
+import com.voxeet.sdk.utils.Map;
+import com.voxeet.sdk.views.VideoView;
+import com.voxeet.uxkit.common.UXKitLogger;
+import com.voxeet.uxkit.common.logging.ShortLogger;
+import com.voxeet.uxkit.presentation.view.AbstractMediaPlayerView;
+
+import org.greenrobot.eventbus.EventBus;
+import org.greenrobot.eventbus.Subscribe;
+import org.webrtc.RendererCommon;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Simple implementation to help integrate screenshare playback into Apps
+ */
+public class ScreenShareMediaPresentationView extends AbstractMediaPlayerView {
+
+    private final static ShortLogger Log = UXKitLogger.createLogger(ScreenShareMediaPresentationView.class);
+    private final ScreenShareEvents events = new ScreenShareEvents();
+
+
+    @Nullable
+    private VideoView videoView;
+
+    @SuppressLint("ClickableViewAccessibility")
+    public ScreenShareMediaPresentationView(@NonNull Context context) {
+        super(context);
+
+
+        ScaleGestureDetector pinchDetector = new ScaleGestureDetector(getContext(), new ScaleGestureDetector.SimpleOnScaleGestureListener() {
```

Comment:
We should extract it to inner class. It is too big to sit in a constructor.

Reply:
Yup did it in the showcase app but didn't bring it here